gobblin-modules/gobblin-orc/src/main/java/org/apache/gobblin/writer/GobblinBaseOrcWriter.java

```diff
@@ -258,7 +261,18 @@ public void close()
   public void commit()
       throws IOException {
     closeInternal();
+    if(this.validateORCDuringCommit) {
+      try {
+        OrcFile.createReader(this.stagingFile, new OrcFile.ReaderOptions(conf));
+      } catch (Exception e) {
+        log.error("Found error when validating ORC file during commit phase", e);
+        HadoopUtils.deletePath(this.fs, this.stagingFile, false);
+        log.error("Delete the malformed ORC file after close the writer: {}", this.stagingFile);
```

Comment:
Isn't this an 'info' log?

Reply:
Also make the statement past tense.

Reply:
This is more an error situation because ideally it shouldn't happen, so we need to delete the file and terminate the commit/ingestion

Reply:
Line 269 is error message, agreed. 
This is info because you are giving information that you have deleted the malformed file.

Reply:
yes, it's an info as we delete the file. But we can log the message at the ERROR level in case user disabled for info level, as this is critical information we'd like it to speak loud.

Reply:
if thats the case you can modify the error message to be like at line 269. 

"Found error when validating ORC file during commit phase. Deleting the malformed ORC file and closing the writer."

The tense of the statement is important in logs to understand whats being done and what is already done.